src/app/pages/home/footer/footer.component.html

```diff
@@ -1 +1,31 @@
-<div>Footer Page</div>
\ No newline at end of file
+
+<nav class=" navbar navbar-dark bg-dark mt-10 clearfix">
+
+    <div class="navbar-expand m-auto navbar-text">
+        Made with
+        <fa-icon [icon] = "heart" class = "hrt" ></fa-icon>
+        by NTUOSS
+        
+    </div>    
+    <div class="navbar-expand m-auto navbar-text ">
+        Follow Us On Social Media
+        <a href= "https://www.facebook.com/ntuoss/?ref=br_rs" target="_blank">
+        <button type = "button" class = "btn">
+            <fa-icon [icon] = "fb"> </fa-icon>      
+        </button>
+        </a>
+        <div class="divider">
+            
+        </div>
+        
+        <a href= "https://twitter.com/ntuoss" target="_blank">
+        <button type = "button" class = "btn" margin = 2.5em>

```

Comment:
I think using the attribute `margin` doesn't achieve anything in this case. If you want spaces between the icon buttons, please consider using the class `ml-1` here which sets some `margin-left` value. More info regarding these utility classes [here](https://getbootstrap.com/docs/4.1/utilities/spacing/#notation).

```suggestion
        <button type="button" class="btn ml-1">
```